_posts/2017-09-25-conference-2017.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: posts
+title: Bazel Conference 2017
+---
+
+Bazel team is pleased to announce our first annual [**Bazel Conference**](https://sites.google.com/corp/bazel.build/conference2017), focused on the needs of our community, with user stories and feedback, migration talks, roadmap, hands-on and break-out tech sessions with Bazel engineers, contributors and users.
```

Comment:
Split into two sentences: "...focused on the needs of our community. The conference will feature user stories..."